tangelo/linq/tests/test_symbolic_simulator.py

```diff
@@ -0,0 +1,65 @@
+# Copyright 2023 Good Chemistry Company.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#   http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+"""A test class to check that the simulator class functionalities are behaving
+as expected for the symbolic backend.
+"""
+
+import unittest
+
+from tangelo.linq import Gate, Circuit
+from tangelo.helpers.utils import installed_backends
+
+from tangelo.helpers.utils import installed_backends
+from tangelo.linq.target.target_sympy import SympySimulator
+
+
+class TestSymbolicSimulate(unittest.TestCase):
+
+    @unittest.skipIf("sympy" not in installed_backends, "Test Skipped: Backend not available \n")
```

Comment:
It would be more useful for that error message to say what the unavailable backend is